plugins.txt

```diff
@@ -1,137 +1,119 @@
-ace-editor:1.1
-ansicolor:0.5.2
-ant:1.7
 antisamy-markup-formatter:1.5
-apache-httpcomponents-client-4-api:4.5.3-2.0
+apache-httpcomponents-client-4-api:4.5.3-2.1
 async-http-client:1.9.40.0
 authentication-tokens:1.3
-blueocean-autofavorite:1.2.1
-blueocean-bitbucket-pipeline:1.3.4
-blueocean-commons:1.3.4
-blueocean-config:1.3.4
-blueocean-dashboard:1.3.4
+blueocean-autofavorite:1.2.2
+blueocean-commons:1.4.2
+blueocean-config:1.4.2
+blueocean-dashboard:1.4.2
 blueocean-display-url:2.2.0
-blueocean-events:1.3.4
-blueocean-git-pipeline:1.3.4
-blueocean-github-pipeline:1.3.4
-blueocean-i18n:1.3.4
-blueocean-jira:1.3.4
-blueocean-jwt:1.3.4
-blueocean-personalization:1.3.4
-blueocean-pipeline-api-impl:1.3.4
-blueocean-pipeline-editor:1.3.4
-blueocean-pipeline-scm-api:1.3.4
-blueocean-rest-impl:1.3.4
-blueocean-rest:1.3.4
-blueocean-web:1.3.4
-blueocean:1.3.4
+blueocean-events:1.4.2
+blueocean-git-pipeline:1.4.2
+blueocean-github-pipeline:1.4.2
+blueocean-i18n:1.4.2
+blueocean-jwt:1.4.2
+blueocean-personalization:1.4.2
+blueocean-pipeline-api-impl:1.4.2
+blueocean-pipeline-editor:1.4.2
+blueocean-pipeline-scm-api:1.4.2
+blueocean-rest-impl:1.4.2
+blueocean-rest:1.4.2
+blueocean-web:1.4.2
+blueocean:1.4.2
 bouncycastle-api:2.16.2
-branch-api:2.0.16
+branch-api:2.0.18
 build-timeout:1.19
-cloudbees-bitbucket-branch-source:2.2.7
-cloudbees-folder:6.2.1
-config-file-provider:2.16.4
-credentials-binding:1.13
+cloudbees-folder:6.3
+config-file-provider:2.18
+credentials-binding:1.15
 credentials:2.1.16
 display-url-api:2.2.0
-docker-commons:1.9
-docker-workflow:1.14
-dockerhub-notification:2.2.0
-durable-task:1.17
+docker-commons:1.11
+docker-workflow:1.15.1
+dockerhub-notification:2.2.1
+durable-task:1.21
 email-ext:2.61
 embeddable-build-status:1.9
 external-monitor-job:1.7
 favorite:2.3.1
-ghprb:1.39.0
-git-client:2.6.0
+ghprb:1.40.0
+git-client:2.7.1
 git-server:1.7
-git:3.6.4
+git:3.8.0
 github-api:1.90
-github-branch-source:2.3.1
+github-branch-source:2.3.2
 github-issues:1.2.4
-github-oauth:0.28.1
+github-oauth:0.29
 github-organization-folder:1.6
 github-pr-coverage-status:1.9.1
-github-pullrequest:0.1.0-rc27
-github:1.28.1
-gitlab-merge-request-jenkins:2.0.0
-gitlab-oauth:1.0.9
-gitlab-plugin:1.5.2
-google-login:1.3
+github-pullrequest:0.1.0-rc29
+github:1.29.0
 gradle:1.28
-gravatar:2.1
-handlebars:1.1.1
 htmlpublisher:1.14
 icon-shim:2.0.3
-jackson2-api:2.8.7.0
-jira:2.5
+jackson2-api:2.8.11.1
 jquery-detached:1.2.1
 jsch:0.1.54.1
-junit:1.23
+junit:1.24
 kerberos-sso:1.4
-kubernetes:1.1.2
-ldap:1.18
-lockable-resources:2.1
+kubernetes:1.3.2
+lockable-resources:2.2
 mailer:1.20
 mapdb-api:1.0.9.0
 mask-passwords:2.10.1
 matrix-auth:2.2
 matrix-project:1.12
```

Comment:
Not sure if we will ever need that?

Reply:
No idea, I have just tried to remove what I know we really don't need it. In this case we might leave it as is since matrix builds might be useful some day

Reply:
One day... so YAGNI. I would opt for dropping it. Also not so sure about `matrix-auth` plugin.

Reply:
@bartoszmajsak matrix-auth is used by both github-auth and openshift-login-plugin

Reply:
Thanks @sthaha - maybe worth to group those plugins together and put some comment?

Reply:
But then maybe we drop `matrix-project` plugin? WDYT guys?